node/source.sls

```diff
@@ -2,6 +2,18 @@
 {% set version = node.get('version', '0.8.20') -%}
 {% set checksum = node.get('checksum', 'b780f58f0e3bc43d2380d4a935f2b45350783b37') -%}
 {% set make_jobs = node.get('make_jobs', '1') -%}
+
```

Comment:
you'd have better luck using the redhat/debian equivalents of `build-essential` instead of keeping all that stuff here

also checkinstall is used later and isn't installed on RedHat

Lastly are you sure openssl-devel is needed?